src/utils/emojis.js

```diff
@@ -85,7 +85,13 @@ const EMOJI_MAP = new Map([
   ['forest', ':evergreen_tree:'],
   ['tree', ':evergreen_tree:'],
   ['hug', ':hugging_face:'],
-  ['hugging', ':hugging_face:']
+  ['hugging', ':hugging_face:'],
+  ['prison', ':gun:'],
+  ['lucky', ':raised_hands:'],
+  ['thunder', ':thunder_cloud_and_rain:'],
+  ['storm', ':thunder_cloud_and_rain:'],
+  ['money', ':money_mouth_face:'],
+  ['dollar', ':money_mouth_face:'],
```

Comment:
The trailing comma is causing linting to fail. Feel free to push another commit to this branch to fix it; no need to open a new PR!